components/settings/settings.cpp

```diff
@@ -414,6 +414,12 @@ void Manager::setBool(const std::string &setting, const std::string &category, c
     setString(setting, category, value ? "true" : "false");
 }
 
+void Manager::apply(const std::string &setting, const std::string &category)
```

Comment:
I guess I should have said it but such a function name is misleading. It doesn't actually apply anything, it removes pending setting changes from the tracker vector. I'm not sure what could be a better name, though.

Reply:
The same thing for existing "apply" function - it does not apply anything too.